src/coreclr/jit/lsraarm64.cpp

```diff
@@ -326,12 +326,19 @@ int LinearScan::BuildNode(GenTree* tree)
 
         case GT_INTRINSIC:
         {
-            noway_assert((tree->AsIntrinsic()->gtIntrinsicName == NI_System_Math_Abs) ||
-                         (tree->AsIntrinsic()->gtIntrinsicName == NI_System_Math_Ceiling) ||
-                         (tree->AsIntrinsic()->gtIntrinsicName == NI_System_Math_Floor) ||
-                         (tree->AsIntrinsic()->gtIntrinsicName == NI_System_Math_Truncate) ||
-                         (tree->AsIntrinsic()->gtIntrinsicName == NI_System_Math_Round) ||
-                         (tree->AsIntrinsic()->gtIntrinsicName == NI_System_Math_Sqrt));
+            NamedIntrinsic ni = tree->AsIntrinsic()->gtIntrinsicName;
+
+            if ((ni == NI_System_Math_Max) || (ni == NI_System_Math_Min))
```

Comment:
It would probably be good to either turn this into a `switch` with a `default: unreached();` or to just swap based on the operand count and assert the IDs